### ISSUE TITLE
Same and cross origin requirements not specified.

### ISSUE THREAD

**Issue by skynavga** [w3c/imsc#158]
Since IMSC allows references to external images, the same and cross origin requirements/constraints need to be defined.

**Comment by palemieux** [comment]
Blocked by https://github.com/w3c/ttml2/issues/405

**Comment by palemieux** [comment]
Duplicate of #280
